lib/ansible/modules/system/sysctl.py

```diff
@@ -374,6 +374,11 @@ def write_sysctl(self):
 # main
 
 def main():
+    conf_file = '/etc/sysctl.conf'
+    if not os.path.isfile(conf_file) and os.path.isdir('/etc/sysctl.d'):
+        # Arch Linux and Manjaro system requires sysctl conf file
+        # inside /etc/sysctl.d directory
+        conf_file = '/etc/sysctl.d/99-ansible-sysctl.conf'
```

Comment:
We might want a 'list of targets'  as there are more variances (/usr/local/etc/sysctl.conf being another)  and to add new option for naming when sysctl.d is there, not everyone will want `99-ansible` .